VERSION

```diff
@@ -1 +1 @@
-0.2.0-dev
+0.1.1
```

Comment:
I was preparing to release generate_file, that is why it was 0.2.0-dev, sorry for the confusion